.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
```

Comment:
I dont mind remove the semi-colons... since wix-code, I got used to cleaner code :)